File: src/nl/rubensten/texifyidea/psi/LatexTokenType.java
package nl.rubensten.texifyidea.psi;

import com.intellij.psi.tree.IElementType;
import nl.rubensten.texifyidea.LatexLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

/**
 * @author Sten Wessel
 */
public class LatexTokenType extends IElementType {
    public LatexTokenType(@NotNull @NonNls String debugName) {
        super(debugName, LatexLanguage.INSTANCE);
    }
}

File: src/nl/rubensten/texifyidea/psi/LatexTokenType.kt
package nl.rubensten.texifyidea.psi

import com.intellij.psi.tree.IElementType
import nl.rubensten.texifyidea.LatexLanguage
import org.jetbrains.annotations.NonNls

/**
 * @author Sten Wessel
 */
class LatexTokenType(@NonNls debugName: String) : IElementType(debugName, LatexLanguage.INSTANCE) {

    override fun toString() = "LatexTokenType." + super.toString()
}